Sources/RedCat/CombineStore.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  CombineStore.swift
+//  
+//
+//  Created by Markus Pfeifer on 15.05.21.
+//
+
+import Foundation
+
+public final class StoreObjectWillChangePublisher {
+	private var observers: [UUID: () -> Void] = [:]
+	
+	public func send() {
+		observers.forEach { $0.value() }
+	}
+	
+	func subscribe(_ subscriber: @escaping () -> Void) -> StoreUnsubscriber {
+		let id = UUID()
+		observers[id] = subscriber
+		return StoreUnsubscriber { self.observers[id] = nil }
+	}
+}
+
+#if os(iOS) || os(macOS)
+#if canImport(Combine)
+
+import Combine
+
+@available(OSX 10.15, *)
+@available(iOS 13.0, *)
+extension StoreObjectWillChangePublisher: Publisher {
+	public typealias Output = Void
+	public typealias Failure = Never
+	
+	public func receive<S: Subscriber>(subscriber: S) where Never == S.Failure, Void == S.Input {
+		let unsubscriber = subscribe {
+			_ = subscriber.receive()
+		}
+		subscriber.receive(subscription: StoreSubscription(unsubscriber))
+	}
+}
+
```

Comment:
Ahhhh, nicely done!
I would only ask you here to add a ```public typealias CombineStore = ObservableStore``` whenever Combine is available so the change doesn't become source breaking (as far as I can see, CombineStore is gone). I would like my example projects to still compile :D

Reply:
yes, I will definitely add)